### PR TITLE
PF-2060: Add missing permission that has broken notebook creation.

### PR DESCRIPTION
**Pull request by yuhuyoyo**
example log for the cloud console: 
```
Sep 29 18:22:46 testnotebook4 c2d-startup[431]: Registering with Proxy...
Sep 29 18:22:46 testnotebook4 c2d-startup[431]: Proxy URL from the config: https://us-central1.notebooks.cloud.google.com/tun/m/4592f092208ecc84946b8f8f8016274df1b36a14
Sep 29 18:23:08 testnotebook4 c2d-startup[431]: ERROR: (gcloud.compute.instances.describe) Could not fetch resource:
Sep 29 18:23:08 testnotebook4 c2d-startup[431]:  - Required 'compute.instances.get' permission for 'projects/terra-vdevel-clean-celery-1754/zones/us-central1-a/instances/testnotebook4'
Sep 29 18:23:08 testnotebook4 c2d-startup[431]: Proxy Registration failed. Unable to get Service Account from VM
```

Missing permission `compute.instances.get`. I tried adding it at the resource level but it didn't work so I added the permission at the project level.

### CODE REVIEW

**Comment by ddietterich** [comment]
Good find! Did that get dropped or is it a change?

**Comment by yuhuyoyo** [comment]
> Good find! Did that get dropped or is it a change?

it's a change. 

but I'm slightly confused. I tested it out in the verily devel env as an admin by adding this permission to the custom role and it works. But the nightly is still failing running against this branch so I still need to confirm if this is the only one missing.

**Comment by yuhuyoyo** [comment]
oh nvm i tested it by adding to the project level.

**Comment by cbookg** [comment]
> > Good find! Did that get dropped or is it a change?
> 
> it's a change.
> 
> but I'm slightly confused. I tested it out in the verily devel env as an admin by adding this permission to the custom role and it works. But the nightly is still failing running against this branch so I still need to confirm if this is the only one missing.

I don't understand how this isn't an API change by Cloud  that would break many customers.

**Comment by yuhuyoyo** [comment]
> > > Good find! Did that get dropped or is it a change?
> > 
> > 
> > it's a change.
> > but I'm slightly confused. I tested it out in the verily devel env as an admin by adding this permission to the custom role and it works. But the nightly is still failing running against this branch so I still need to confirm if this is the only one missing.
> 
> I don't understand how this isn't an API change by Cloud that would break many customers.

normally if you just have a pre-defined gcp iam role assigned to, you won't get affected. so i guess not many people have custom role like us. There is a similar bug when i filed it though so I don't think we are the only one

**Review by zloery — APPROVED** [review]

File: service/src/main/java/bio/terra/workspace/service/workspace/CloudSyncRoleMapping.java [hunk 1]
```diff
@@ -71,6 +71,7 @@ public class CloudSyncRoleMapping {
               "artifactregistry.tags.update",
               "cloudbuild.builds.create",
               "cloudbuild.builds.update",
+              "compute.instances.get",
```
Comment (zloery) [inline service/src/main/java/bio/terra/workspace/service/workspace/CloudSyncRoleMapping.java:74]: I think this just gives read access to metadata and not instance data which is fine, but we should test once this is merged

**Pull request merged**